firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2249,13 +2253,37 @@ extension BrowserViewController: LegacyTabDelegate {
         tab.addContentScript(readerMode, name: ReaderMode.name())
 
         // only add the logins helper if the tab is not a private browsing tab
-        if !tab.isPrivate {
+        if !tab.isPrivate, autofillLoginNimbusFeatureFlag() {
```

Comment:
@razvanlitianu since the feature flag has changed for credit card, did we update that too?

Reply:
It didn't really change. Even tho the file name was changed the code is still compiled in one big swift file and the generated code is still the same.